openedx/features/content_tagging/rest_api/v1/views.py

```diff
@@ -0,0 +1,76 @@
+"""
```

Comment:
Same suggestions here as on https://github.com/open-craft/openedx-learning/pull/4/files/a0fedd36bbe06ef0a4c00466c8c9570f3c35616d -- we can get by with far less code here.